ui/nuxeo-user-group-management/nuxeo-user-group-permissions-table.js

```diff
@@ -163,7 +168,7 @@ import '../nuxeo-button-styles.js';
               <dom-repeat items="[[documents]]" as="document">
                 <template>
                   <div class="table-row">
-                    <div class="flex-3">
+                    <div class="flex-3 document-name">
```

Comment:
Since we are displaying the title of the document, a more explicit name for this class should be:
```suggestion
                    <div class="flex-3 title">
```